src/main/java/org/opentripplanner/updater/trip/TimetableSnapshotSource.java

```diff
@@ -493,22 +495,44 @@ private Result<?, UpdateError> validateAndHandleAddedTrip(
       return UpdateError.result(tripId, NO_START_DATE);
     }
 
-    // Check whether at least two stop updates exist
-    if (tripUpdate.getStopTimeUpdateCount() < 2) {
-      debug(tripId, "ADDED trip has fewer than two stops, skipping.");
+    final List<StopTimeUpdate> stopTimeUpdates = removeUnknownStops(tripUpdate, tripId);
```

Comment:
Maybe raise the point in dev meeting? 
I'm happy with OTP fixing the RT data in order to accept it whenever possible but I feel that wasn't always a shared opinion as it might indicate an issue the data provider has to fix/handle.
At least the debug line could be more pro-eminent, like an actual warning/error? Also potentially we could attach a Journey Alert to the trip ("Also stops in: X,Y,Z") in order to inform the end-user that this journey has additional stops?

Reply:
I've now added warnings to the `UpdateResult`. They will end up in the log and the micrometer metrics.